### PR TITLE
data/saadali05.json

### PR DESCRIPTION
**Pull request by saadali05**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8752"></a>

### CODE REVIEW

File: saadali.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Saad Ali",
+  "bio": "Aspiring code explorer, embarking on a journey of learning and creating. Turning curiosity into code, one step at a time. ",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/saadali05",
+      "icon": "FaGithub "
+    }
```
Comment (sital002) [inline saadali.json:9]:
```suggestion
    },
```
Please add comma

File: saadali.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://www.linkedin.com/in/saadali5/",
+      "icon": "FaLinkedin"
+    }
```
Comment (sital002) [inline saadali.json:14]:
```suggestion
    },
```

File: saadali.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      "name": "Follow me on Dev Community",
+      "url": "https://dev.to/saadi5603",
+      "icon": "FaDevcommunity "
+    }
```
Comment (sital002) [inline saadali.json:19]:
```suggestion
    },
```

File: saadali.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      "name": "Follow me on Dev Community",
+      "url": "https://dev.to/saadi5603",
+      "icon": "FaDevcommunity "
+    } 
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SAADI5603",
+      "icon": "FaTwitter"
+    }
```
Comment (sital002) [inline saadali.json:24]:
```suggestion
    },
```

File: saadali.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/saadxalii/",
+      "icon": "FaInstagram"
+    }
```
Comment (sital002) [inline saadali.json:29]:
```suggestion
    },
```

File: saadali.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/profile.php?id=100004491677589",
+      "icon": "FaFacebook"
+    }
```
Comment (sital002) [inline saadali.json:34]:
```suggestion
    },
```

**Review by sital002 — CHANGES_REQUESTED** [review]
Your filename must match your github username and the file must be inside data folder.
Please move the file from saadali.json to data/saadali05.json .

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please move your file to the data folder and make sure that your file name matches your GitHub username exactly. Instead of `saadali.json` it should be `data/saadali05.json`

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart-json to help you update this Pull Request.

**Pull request merged**